frameworks/Python/pyramid/frameworkbenchmarks/views.py

```diff
@@ -21,7 +21,7 @@ def test_2(request):
     """
     Test type 2: Single database query
     """
-    num = randint(1, 10001)
+    num = randint(1, 10000)
```

Comment:
Is `randint` inclusive-inclusive? I feel like this was changed to `randint(1, 10001)` because it's inclusive-exclusive.